### ISSUE TITLE
Why not contribute to meta?

### ISSUE THREAD

**Issue by mateodelnorte** [chimpwizard-wand/wand#1]
Hi @ndruwizz. I noticed a number of your packages are copies of https://github.com/mateodelnorte/meta packages with some find/replace to rename things. You also have some new additions that would be great to have in meta. 

Would you like to help contribute to meta, instead of working on a fork?

**Comment by ndruwizz** [comment]
@mateodelnorte yes. i am still testing the change i made..  since there are couple that are in your core code i didnt want to break anything, so let me send a PR and you can take a look to those critical changes first.

**Comment by ndruwizz** [comment]
It looks like you already fix it. it was somewhere in find-module-bin. Previous version had an issue when trying to resolve packages and if those were installed a global scope were not able to find it. I'll find some time to check if i still can reproduce the issue i had back then with your new code.
